src/main/java/uk/gov/justice/digital/delius/transformers/NsiTransformer.java

```diff
@@ -38,6 +38,7 @@
                 .notes(n.getNotes())
                 .intendedProvider(ProbationAreaTransformer.probationAreaOf(n.getIntendedProvider(), INCLUDE_INTENDED_PROVIDER_TEAMS))
                 .active(zeroOneToBoolean(nsi.getActiveFlag()))
+                .softDeleted(zeroOneToBoolean(nsi.getSoftDeleted()))
```

Comment:
won't this always be false due to the above change in the filter?

Reply:
Filtering on nsi's happen after transformation to api.Nsi